### PR TITLE
Open source: logging

### PR DESCRIPTION
**Pull request by toly-kournik**
* **Problem Overview**  
 re-factor logging library to conform to the open source structure design
* **Testing Done**  
  unittests

### CODE REVIEW

**Comment by WildFireFlum** [comment]
Should probably mark it as [don't auto-bump]

File: logging/CMakeLists.txt [hunk 1]
```diff
@@ -1,26 +0,0 @@
-project(logging  LANGUAGES CXX)
-
-
```
[the remark this answers is not in the document]
Comment (toly-kournik) [reply logging/CMakeLists.txt:3]: I used git mv for everything

[the remark this answers is not in the document]
Comment (toly-kournik) [reply logging/CMakeLists.txt:3]: but this file appears as new :-/

**Review by WildFireFlum — APPROVED** [review]
Please tag the commit with "don't auto bump"

File: bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp [hunk 2]
```diff
@@ -1528,51 +1527,26 @@ void BcStTest::printConfiguration() {
...
+
   // logging::Logger::getInstance("serializable").setLogLevel(logLevel);
   // logging::Logger::getInstance("concord.bft.st.dbdatastore").setLogLevel(logLevel);
   // logging::Logger::getInstance("rocksdb").setLogLevel(logLevel);
-  logging::Logger::getInstance("concord.bft").setLogLevel(logLevel);
-  logging::Logger::getInstance("concord.bft.st.dst").setLogLevel(logLevel);
-  logging::Logger::getInstance("concord.bft.st.src").setLogLevel(logLevel);
-  logging::Logger::getInstance("concord.util.handoff").setLogLevel(logLevel);
-  logging::Logger::getInstance("concord.bft.st.rvb").setLogLevel(logLevel);
+  // logging::Logger::getInstance("concord.bft").setLogLevel(logLevel);
```
Comment (eyalrund) [inline bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp:1545]: what is this?

Comment (toly-kournik) [reply bftengine/tests/bcstatetransfer/bcstatetransfer_tests.cpp:1545]: misuse of the library in test.
it's not critical, but should be either configured from file or two implementations should be aligned in terms of API

**Review by eyalrund — APPROVED** [review]

**Pull request merged**